client/src/components/Page.js

```diff
@@ -111,15 +105,15 @@ export default class Page extends Component {
 		propFilter.push('pageProps')
 		const filteredNextProps = Object.without(this.props, ...propFilter)
 		const filteredProps = Object.without(prevProps, ...propFilter)
-		if (!_isEqualWith(filteredProps, filteredNextProps, this.equalFunction)) {
+		if (!_isEqualWith(filteredProps, filteredNextProps, utils.treatFunctionsAsEqual)) {
 			this.loadData()
 		} else {
 			// Location always has a new key. In order to check whether the location
 			// really changed filter out the key.
 			const locationFilterProps = ['key']
 			const nextPropsFilteredLocation = Object.without(this.props.location, ...locationFilterProps)
 			const propsFilteredLocation = Object.without(prevProps.location, ...locationFilterProps)
-			if (!_isEqualWith(propsFilteredLocation, nextPropsFilteredLocation, this.equalFunction)) {
+			if (!_isEqualWith(propsFilteredLocation, nextPropsFilteredLocation, this.treatFunctionsAsEqual)) {
```

Comment:
shouldn't it be `utils.treatFunctionsAsEqual` ?

Reply:
You're right.